Dockerfile

```diff
@@ -28,6 +28,7 @@ RUN apt-get update && apt-get install -y \
         zlib1g-dev \
         libffi-dev \
         libssl-dev \
+        libmagic-dev \
```

Comment:
You don't need this in `.travis.yml` as well ?

Reply:
It's already installed in travis.